test/standalone/main_return_error/!u8.zig

```diff
@@ -0,0 +1,7 @@
+const Err = error {
```

Comment:
You should also add a test case where main returns a non-zero value.

Reply:
I have added one but it needs something to check that its exit code is correct, do you know how I would do so?

Reply:
Sorry I'm not familiar with the test framework.  I wish I was more familiar.